convoy/batch.py

```diff
@@ -4489,7 +4489,7 @@ def _construct_task(
                         mpi_opts.append('--allow-run-as-root')
                     mpi_command = 'mpirun {} {}'.format(
                         ' '.join(mpi_opts),
-                        util.wrap_commands_in_shell([task.command]))
+                        task.command)
```

Comment:
Can you check the jobs documentation to ensure that there is some text stating that if multiple commands are specified, they should be wrapped in a shell?